juriscraper/pacer/reports.py

```diff
@@ -221,7 +235,7 @@ def download_pdf(
                     f"Failed to get docket entry in case: "
                     f"{pacer_case_id=} at {url}"
                 )
-            if "This document is not available" in r.text:
+            if "document is not available" in r.text:
```

Comment:
Since we don't have tests here, we should add a comment with links to examples, and all the cases it covers. Something like:

```
# See: https://.....example1
# See: https://...example2
# Matches against:
# "This document is not available" and
# "The document is not available"
```